presto-bigquery/src/main/java/io/prestosql/plugin/bigquery/BigQueryMetadata.java

```diff
@@ -118,6 +123,30 @@ public BigQueryMetadata(BigQueryClient bigQueryClient, BigQueryConfig config)
         return tableNames.build();
     }
 
+    @Override
+    public Optional<ConnectorViewDefinition> getView(ConnectorSession session, SchemaTableName viewName)
+    {
+        TableInfo tableInfo = getBigQueryTable(viewName);
+
+        if (tableInfo.getDefinition() instanceof ViewDefinition) {
+            List<BigQueryColumnHandle> columns = getTableColumns(tableInfo);
+            ViewDefinition viewDefinition = tableInfo.getDefinition();
+            ConnectorViewDefinition definition = new ConnectorViewDefinition(
+                    viewDefinition.getQuery(),
+                    Optional.of(catalogName.toString()),
+                    Optional.of(viewName.getSchemaName()),
+                    columns.stream()
+                            .map(column -> new ConnectorViewDefinition.ViewColumn(column.getName(), column.getPrestoType().getTypeId()))
+                            .collect(toImmutableList()),
+                    Optional.ofNullable(tableInfo.getDescription()),
+                    Optional.empty(),
+                    false);
+            viewDefinition.getQuery();
```

Comment:
Is the second call to `viewDefinition.getQuery()` needed?

Reply:
It's my mistake. Let me remove.